sdk/communication/Azure.ResourceManager.Communication/src/autorest.md

```diff
@@ -8,7 +8,7 @@ azure-arm: true
 title: communication
 namespace: Azure.ResourceManager.Communication
 # default tag is a preview version
-require: https://github.com/Azure/azure-rest-api-specs/blob/bbee55edb05169aba6d8d1944c0e2bc2b9408943/specification/communication/resource-manager/readme.md
+require: https://github.com/Azure/azure-rest-api-specs/blob/d71132a3e8bf8ad22ab77991805e033ed1b66dff/specification/communication/resource-manager/readme.md
```

Comment:
Please specify the expected release date in the changelog.md file, release the `Unreleased` with the actual date, the format is `yyyy-mm-dd`